benches/decoder_benchmark.rs

```diff
@@ -0,0 +1,56 @@
+use criterion::*;
+use std::time::Duration;
+
+use rand::distributions::Alphanumeric;
+use rand::{thread_rng, Rng};
+use fountaincode::decoder::Decoder;
+use fountaincode::encoder::Encoder;
+use fountaincode::types::*;
+
+fn encode_decode_random(total_len: usize, chunk_len: usize) {
+    let s: String = thread_rng()
```

Comment:
If I'm following the code correctly, the the benchmark stats will include the time it takes to generate the random message.